dwave/gate/primitives.py

```diff
@@ -12,6 +12,10 @@
 #    See the License for the specific language governing permissions and
 #    limitations under the License.
 
+"""Primitive types used with quantum circuits.
+
+Contains primitive types such as qubits, bits and variables."""
```

Comment:
nit: [`pep8`](https://peps.python.org/pep-0008/#documentation-strings) calls out (funnily enough) this as the most important bit on docstrings :laughing: :
> Note that most importantly, the """ that ends a multiline docstring should be on a line by itself

Reply:
Ha, I recall this now as well! Wouldn't say it's _the_ most important thing though. 😅